.gitignore

```diff
@@ -28,3 +28,5 @@ target/
 
 # turbo
 .turbo/
+
+esbuild-kit
```

Comment:
Is this needed?

Reply:
It keeps getting auto-generated when I build. Asked in dev if someone knows more about it. I'm on windows using wsl.

Reply:
not needed, just some issue in my setup. fixed. removing it now